runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -310,4 +339,98 @@ private void removeGameframe()
 			client.setCompass(compass);
 		}
 	}
+
+	private ArrayList<Widget> getMemberSkills()
+	{
+		ArrayList<Widget> membersOnlyWidgets = new ArrayList<>();
+
+		Widget agility = client.getWidget(WidgetInfo.SKILL_AGILITY),
+				construction = client.getWidget(WidgetInfo.SKILL_CONSTRUCTION),
+				farming = client.getWidget(WidgetInfo.SKILL_FARMING),
+				fletching = client.getWidget(WidgetInfo.SKILL_FLETCHING),
+				herblore = client.getWidget(WidgetInfo.SKILL_HERBLORE),
+				hunter = client.getWidget(WidgetInfo.SKILL_HUNTER),
+				thieving = client.getWidget(WidgetInfo.SKILL_THIEVING),
+				slayer = client.getWidget(WidgetInfo.SKILL_SLAYER);
+
+		membersOnlyWidgets.add(agility);
+		membersOnlyWidgets.add(construction);
+		membersOnlyWidgets.add(farming);
+		membersOnlyWidgets.add(fletching);
+		membersOnlyWidgets.add(herblore);
+		membersOnlyWidgets.add(hunter);
+		membersOnlyWidgets.add(thieving);
+		membersOnlyWidgets.add(slayer);
+
+		return membersOnlyWidgets;
+	}
+
+	private void shuffleSkills(boolean hideMembers)
+	{
+		if (hideMembers && client.getWidget(WidgetInfo.SKILL_TOTAL) != null)
+		{
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeX(WOODCUTTING_X);
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeY(WOODCUTTING_Y);
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeX(client.getWidget(WidgetInfo.SKILL_AGILITY).getRelativeX());
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeY(client.getWidget(WidgetInfo.SKILL_AGILITY).getRelativeY());
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeX(PRAYER_X);
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeY(PRAYER_Y);
+			client.getWidget(WidgetInfo.SKILL_CRAFTING).setRelativeX(client.getWidget(WidgetInfo.SKILL_HERBLORE).getRelativeX());
```

Comment:
Why are some of these swaps using the position of other widgets, and others using hard-coded values?

Reply:
Added comments for clarity. When members skills are hidden, some f2p skills are moved to fill the gaps in the skills tab